### ISSUE TITLE
Please follow semver

### ISSUE THREAD

**Issue by garak** [SimpleBus/SymfonyBridge#75]
I see that latest commits were released as patch versions (e.g. `5.1.5`, `5.1.4`, etc.) despite the fact they contain changes that are not bugfixes.
You should instead follow [semver](https://semver.org/) and release next changes as minor versions (e.g. `5.2.0`).

**Comment by ruudk** [comment]
Did these releases break anything?

**Comment by garak** [comment]
Nothing was broken so far. But that's not the point.

**Comment by cmodijk** [comment]
I agree with @garak we added a new feature and the patch versions should only be used for bug fixes. We should have bumped the minor instead. But it is already done know and we did not brake BC so i think we can close this right?

**Comment by garak** [comment]
Of course, if we all agree, this is issue can be closed

**Comment by cmodijk** [comment]
Thanks @garak and @ruudk !
